src/main/java/ch/njol/skript/util/Date.java

```diff
@@ -67,18 +65,34 @@ public String toString() {
 	}
 	
 	/**
+	 * Get the timestamp of this date

```

Comment:
doesn't really provide any new info that the name of the method and the `@return` didn't already provide

Reply:
While I do agree it does not provide any new information, I also doesn't hurt anything. 
I did this one more for javadocs.

If you truly feel I should remove it let me know and I will.

Reply:
Ok, I added a plus as well as minus method. 
I also added the Date#now() method.